assets/scss/_content.scss

```diff
@@ -0,0 +1,18 @@
+//
+// Style Markdown content
+//
+
+.td-content {
+    order: 1;
+
+    img {
+        @extend .img-fluid;
+        margin-top: 1%;
+        margin-bottom: 1%;
+        padding: 1%;
+        border-style: solid;
+        border-width: 1px;
+        border-color: black
+    }
+
+}
```

Comment:
All the images styling can be done from here

Reply:
Lovely! That was for sure more elegant than what I would have done :)